### DISCUSSION TITLE
当我设置vc-viewer组件的globe属性为false的时候，界面加载不成功，控制台报错，我的期望是不加载地球，加载自定义的3DTitle.

### DISCUSSION THREAD

**Discussion by zenghaiy** [zouyaoji/vue-cesium#545] [Q&A]
![error](https://github.com/zouyaoji/vue-cesium/assets/88429575/ebaccedf-629a-43a8-87d2-c7bd4e1f1ed6)

**Comment by zouyaoji** [comment]
你好，可以确认一下原生viewer支持不？不支持的话vc-viewer也没办法。
